client/src/components/common/applicant-info-1.vue

```diff
@@ -403,10 +404,12 @@ export default class ApplicantInfo1 extends Vue {
     return newReqModule.editMode
   }
   get jurisdictionOptions () {
-    if (this.location === 'IN') {
-      return this.$intJurisdictions.map(jurisdiction => ({ value: jurisdiction.text, text: jurisdiction.text }))
-    }
-    return this.$canJurisdictions.map(jurisdiction => ({ value: jurisdiction.text, text: jurisdiction.text }))
+    const canJurisdiction = this.$canJurisdictions.filter(jur => jur.value !== Location.BC)
+    const intJurisdiction = this.$intJurisdictions.filter(jur => jur.value !== Location.Canadian)
```

Comment:
It's too bad this is doing the filter work every time, but it only uses one of the arrays below.

How about moving this code inside the if-the-else below?

Reply:
Good call, will do!

Reply:
Updated!